### DISCUSSION TITLE
Is it possible to change the default port number?

### DISCUSSION THREAD

**Discussion by ahuckphin** [calcom/docker#65] [General] [answered]
Instead of port 3000 for Calendso, can I change the it to port 80 instead?

**Comment by NanKooDev** [comment, marked as answer]
You can change it in the Dockerfile. You then also need to change it in the .env file. Just look for the number 3000 and change it to whatever port you want, just make sure you use the same port in both files.
You must do this before using the `docker-compose up --build` command.
If it doesn't work, you can try creating the container like normal, and then changing the port afterwards.
